reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,143 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/transactions:
+    get:
+      operationId: get-api-v1-account-transactions
+      summary: Retrieve the authenticated customer's transactions
+      tags:
+        - Account
+        - Transactions
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+        - in: query
+          name: page
+          schema:
+            type: integer
+            minimum: 1
+            default: 1
+          description: The current page
+        - in: query
+          name: size
+          schema:
+            type: integer
+            minimum: 1
+            default: 100
+          description: The page size per page
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: 'Customer transactions list ordered by createdat DESC'
+                type: object
+                properties:
+                  page: 
+                    type: integer
+                  size:
+                    type: integer
+                  collection:
```

Comment:
Do we wanna call this property `transactions` for clarity?

Reply:
i wanna be more generic because endpoint already is account/transactions maybe 'list' 'items' What do you mean?

Reply:
I mean looking at the example response, it feels like we need a name that gives more information about what it is, i.e. giving a name to the collection/ list/ items:
```
{
  "page": 1,
  "size": 100,
  "transactions": [
    {
      "CreatedAt": "2020-03-24T17:32:28Z",
      "Description": "SCA Weekly Pass  24 Mar 2020 to 31 Mar 2020",
      "Provider": "PayPal",
      "Reference": "customer@gmail.com",
      "CurrencyCode": "EUR",
      "Amount": -0.99
    },
    ...
   ]
}   
```

I'm thinking from the point of view of someone using this endpoint, IMO `response.transactions` reads better than `response.collection`